### DISCUSSION TITLE
Pending requests with azure WAF

### DISCUSSION THREAD

**Discussion by antoinechamot** [axios/axios#6456] [Q&A]
I'm encountering an issue where I make a POST request using axios, but it gets blocked by Azure WAF, resulting in a 403 status code. Following this, any subsequent requests I attempt to make with axios do not proceed and are indefinitely stuck in a pending state. 
It appears as though the connection is perpetually frozen on the request that Azure rejected. 
What could be causing this, and how can I avoid this situation?

**Comment by HAMMUODI2021** [comment]
F
